03-blog-preview/style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  font-family: "Figtree", sans-serif;
+  font-optical-sizing: auto;
+  font-style: normal;
+  font-size: 16px;
+  transition: all 0.2s;
+}
+.yellow-background {
+  background-color: hsl(47, 88%, 63%);
+}
+.black-text {
+  width: fit-content;
+  padding: 10px;
+  border-radius: 5px;
+}
+.card {
+  background-color: white;
+  border: 0.1rem solid hsl(0, 0%, 7%);
+  border-radius: 20px;
+  display: flexbox;
+  justify-content: center;
+  max-width: 350px;
+  margin: 250px auto;
```

Comment:
It's best if parent controls the positioning. Let's discuss why in person. One huge disadvantage is that now this card can only be positioned in the center and it must have 250px top and bottom margins (generates scrollbar on smaller screens).